lib/Source/AutoHog/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# AutoHoG importer library
+find_package(rapidjson)
```

Comment:
Since the RapidJSON dependency probably isn't going to be updated frequently, I think it's save to already include it in this PR with a hardcoded git commit.

```suggestion
include(ExternalProject)

# Download RapidJSON
# TODO (#818): automatically extract the commit from the bazel WORKSPACE file 
ExternalProject_Add(
    rapidjson
    PREFIX "vendor/rapidjson"
    GIT_REPOSITORY "https://github.com/Tencent/rapidjson.git"
    GIT_TAG f54b0e47a08782a6131cc3d60f94d038fa6e0a51
    TIMEOUT 10
    CMAKE_ARGS
        -DRAPIDJSON_BUILD_TESTS=OFF
        -DRAPIDJSON_BUILD_DOC=OFF
        -DRAPIDJSON_BUILD_EXAMPLES=OFF
    CONFIGURE_COMMAND ""
    BUILD_COMMAND ""
    INSTALL_COMMAND ""
    UPDATE_COMMAND ""
)

# Prepare RapidJSON (RapidJSON is a header-only library)
ExternalProject_Get_Property(rapidjson source_dir)
set(RAPIDJSON_INCLUDE_DIR ${source_dir}/include)
```